tests/lib/ocpp/v201/test_smart_charging_handler.cpp

```diff
@@ -320,6 +340,56 @@ TEST_F(ChargepointTestFixtureV201, K01FR35_IfChargingSchedulePeriodsAreNotInChon
     EXPECT_THAT(sut, testing::Eq(ProfileValidationResultEnum::ChargingSchedulePeriodsOutOfOrder));
 }
 
+TEST_F(ChargepointTestFixtureV201, ValidateChargingStationMaxProfile_NotChargingStationMaxProfile_Invalid) {
```

Comment:
You need to fix the test name here to match the convention

Reply:
Updated. This specific test is a general validation that doesn't have a specific FR.XX value.